MRN-Code/coinstac-ui
# Refactor main and render boot processes

Asana task:

* **Problem:** This started as a fix for non-appearing UI notifications, but led to some clean-up and clarification of the application’s boot process.
* **Solution:**
  * Update boot files: move `require` calls to the top where possible and update code style
  * Move `trace` and `process`-event error handling to the main process
  * Simplify boot steps and the manner in which they’re called
* **Testing:**
  * Pull down branch and attempt to boot the app.
  * Add a notification via the console. Confirm it actually appears.

I’d be open to testing the boot process, too. Let me know if I should do that along with this.
